tests/test_transactions.py

```diff
@@ -61,3 +61,29 @@ def test_rinkeby_send(yield_dongle):
     assert signed.v in [(CHAIN_ID * 2 + 35) + x for x in (0, 1)]
     assert signed.r
     assert signed.s
+
+
+def test_eip1559_send(yield_dongle):
+    """Test a type 2 (EIP-1559) transaction"""
+    CHAIN_ID = 3
+    bribe_per_gas = int(1e9)
+    max_fee_per_gas = int(20e9)
+
+    # Matches tx from app-ethereum tests
+    signed = create_transaction(
+        to="0xb2bb2b958afa2e96dab3f3ce7162b87daea39017",
```

Comment:
How does checksum address play into this?

Does the dongle not care either way?

Would another backlog item be to make use of checksum addresses for safety reasons for the users of this lib?

Reply:
Checksum addresses are for validity checking for UX (to prevent copy errors and the like).  Decoding hex to bytes, `0x0A` and `0x0a` are equivalent:

    >>> from eth_utils import decode_hex
    >>> decode_hex('0x0a')
    b'\n'
    >>> decode_hex('0x0A')
    b'\n'

But a pretty good feature request might be adding checksum validation to user-facing helper functions.